src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -150,147 +154,235 @@ internal partial class Session
                         continue;
                     }
 
-                    var filterText = model.GetCurrentTextInSnapshot(currentItem.Item.Span, textSnapshot, textSpanToText);
+                    // Check if the item matches the filter text typed so far.  Note: we completely
+                    // handle the deletion case and use a very weak 'prefix' match approach in order
+                    // to not accidently match random parts of a completion item as you delete some
+                    // existing code.
                     var matchesFilterText = MatchesFilterText(helper, currentItem.Item, filterText, model.Trigger, filterReason, recentItems);
```

Comment:
The comment seems like it's a detail of `MatchesFilterText`, not applicable here.